src/Entity/Sylius/Order.php

```diff
@@ -1000,9 +1000,10 @@ public function getFulfillmentMethod(): string
     public function getPaymentMethod(): string
     {
         $payment = $this->getLastPayment();
+        $method = $payment->getMethod();
```

Comment:
I had to revert this change, because sometimes `$payment` can be null

Reply:
Oh ok, sorry. I made this change because some tests are failing because the methods are null and then when try to acccess getCode() method an exception is raised.

Reply:
Ok thanks I think it's more a problem of test data not properly initialized